packages/@react-stately/selection/src/SelectionManager.ts

```diff
@@ -494,4 +494,8 @@ export class SelectionManager implements MultipleSelectionManager {
   isLink(key: Key) {
     return !!this.collection.getItem(key)?.props?.href;
   }
+
+  getItemProps(key: Key) {
```

Comment:
Adding this to selection manager feels a bit weird, but not sure where else to put it. `useSelectableCollection` and `useSelectableItem` don't currently receive the collection as props, except via `SelectionManager`...

Reply:
agreed, it's a bit odd, however, we already have 'isLink' in here, which is almost the same thing and is only tangentially related to selection

would it make more sense to just return the collection and then handle this specific logic outside?

Reply:
well, the original idea was that the `MultipleSelectionManager` interface could have multiple implementations, perhaps not even using collections under the hood. So if we expose the collection from the interface that will force other implementations to also use collections.

Reply:
that makes sense :-/